grafana/dashboards/statistics.json

```diff
@@ -696,16 +748,16 @@
           "value": "GMT"
         },
         "datasource": "TeslaMate",
-        "definition": "select name from pg_timezone_names where name like 'posix/%';",
+        "definition": "SELECT current_setting('TIMEZONE');",
```

Comment:
Subsequently changing the timezone via `PGTZ` does not work for me. 

Could both approaches be combined? Then a meaningful timezone could be preselected, but one would still have the possibility to change the timezone.

Reply:
> Subsequently changing the timezone via `PGTZ` does not work for me.
> 
You have set it for the Grafana container?

> Could both approaches be combined? Then a meaningful timezone could be preselected, but one would still have the possibility to change the timezone.
Well, "select current_setting('TIMEZONE'), name from pg_timezone_names where name like 'posix/%';" should do it. It does for my changed version, it does not for the "official" version. Whyoever...

Reply:
> You have set it for the Grafana container? (Not the Database one!)

Oh, that worked! Okay, let's make the time zone configurable then but default to the current time zone setting.

Reply:
Ok, that's fine... And links as soon as the hidden fiels issue is fixed?

Reply:
Yeah, that'd be great. I hope they will fix it soon!

Reply:
Just saw, you have the links in. So, then it is only needed to hide the two fields, when it is fixed.

Reply:
Ah, sorry that's what I meant. Those almost invisible fields are fine for now.